internal/api/http/httputils/routes_utils.go

```diff
@@ -46,3 +48,25 @@ func ReturnSuccess(w http.ResponseWriter, statusCode int, data any) {
 		return
 	}
 }
+
+func SetDefaultQueryParams(query *url.Values, field string) {
+	if query.Get("limit") == "" {
+		query.Set("limit", DefaultPaginationLimitStr)
+	}
+	if query.Get("offset") == "" {
+		query.Set("offset", DefaultPaginationOffsetStr)
+	}
+	if query.Get("sort") == "" {
+		sort := fmt.Sprintf("%s:%s", field, DefaultSortOrder)
+		query.Set("sort", sort)
+	}
+}
+
+func GetQueryParams(query *url.Values, field string) map[string][]string {
```

Comment:
Why is it string -> []string, its just query -> value, it does not need to be and array. I don'y like the idea of passing same query multiple times. Lets not allow this. If we get same query multiple time, return appropriate error, and cancel the request.